example/index.html

```diff
@@ -6,6 +6,7 @@
   <title>This is an example</title>
 
   <link href='https://fonts.googleapis.com/css?family=Roboto:300,400,500' rel='stylesheet' type='text/css'>
+  <script src="https://rawgit.com/airbnb/polyglot.js/master/build/polyglot.js" type="text/javascript"></script>
```

Comment:
I know what you mean, but the version installed by npm is NOT frontend version of polyglot. Try it, and you'll see ...